v2/pages/userInstanceQuestionExam/userInstanceQuestionExam.ejs

```diff
@@ -47,6 +47,34 @@
               </div>
             </div>
             <% } %>
+            
+            <% if (showAllSubmissions) { %>
+            <div class="panel-group">
+                <% for (sNum = 0; sNum < allSubmissions.length; sNum++) { 
+                thisSubmission = allSubmissions[sNum]; 
+                if (thisSubmission.correct === null) {
+                    panelType = "panel-info"; 
+                    submissionLabel = "<span class='label label-primary'>saved, not graded</span>"; 
+                }
+                else if (thisSubmission.correct) {
+                    panelType = "panel-success";
+                    submissionLabel = "<span class='label label-success'>correct</span>";
+                }
+                else {
+                    panelType = "panel-danger";
+                    submissionLabel = "<span class='label label-danger'>incorrect</span>";
+                } %>
+                <div class="panel <%= panelType %> submission-block">
+                    <div class="panel-heading">
+                        <h3 class="panel-title">Submission <%= allSubmissions.length - sNum %> <%- submissionLabel %></h3>
```

Comment:
Can we also add submission time here? Use the SQL functions `sprocs/format_date_*.sql` to format.